db/models.py

```diff
@@ -1,4 +1,11 @@
+from django.contrib.auth.models import AbstractUser
 from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class User(AbstractUser):
+    first_name = models.CharField(max_length=255, blank=True, null=True)
+    last_name = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Allowing 'first_name' and 'last_name' to be null might lead to inconsistencies if other parts of the application assume these fields are always strings. Consider whether this change aligns with the application's requirements.